dpe/src/dpe_instance.rs

```diff
@@ -192,7 +193,7 @@ impl<C: Crypto> DpeInstance<'_, C> {
 }
 
 #[repr(transparent)]
-#[derive(Copy, Clone)]
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
```

Comment:
I wonder if we should derive these traits for test config only. Just want to avoid the extra code getting linked if someone inadvertently uses these traits in runtime fw.

Reply:
So, I had to add these derived traits to `TciMeasurement` so that the `GetTaggedTciResp` could also derive them. What would be the problem with this code getting linked if someone uses it, exactly?